modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/Validator.scala

```diff
@@ -31,13 +29,10 @@ object Validator {
 
   private def enoughPeersForConsensus[F[_]: Monad](
     clusterStorage: ClusterStorage[F],
-    collateral: Collateral[F],
     peersCount: PosInt
   ): F[Boolean] =
     clusterStorage.getPeers
       .map(_.filter(p => isReadyForBlockConsensus(p.state)))
-      .map(_.toList)
-      .flatMap(_.filterA(p => collateral.hasCollateral(p.id)))
```

Comment:
I think that the idea is that `ClusterStorage` holds peers only with collateral, therefore when you call `getPeers` you don't need to verify it additionally.

Reply:
but here we verify block's creators. we don't even need to know those peers, so I'd suggest to keep the validation, because block is invalid if it's created by peer with no collateral 🤔

Reply:
yet, it's a context validation because we need to know the balance of the peer

Reply:
ah, I misread that part. indeed it's correct here

Reply:
I restored verification of collateral, though I also think it is redundant here.